packages/commons/src/config/jwtConfig.ts

```diff
@@ -0,0 +1,20 @@
+import { z } from "zod";
+
+export const JWTSeedConfig = z
+  .object({
+    JWT_SUBJECT: z.string(),
+    JWT_AUDIENCE: z.string().transform((v) => v.split(",")),
+    JWT_ISSUER: z.string(),
+    JWT_SECONDS_TO_EXPIRE: z.coerce.number(),
```

Comment:
if we keep the configs here, I would suggest this rename. It would be clearer what they are used for when listing env vars
```suggestion
    GENERATED_JWT_SUBJECT: z.string(),
    GENERATED_JWT_AUDIENCE: z.string().transform((v) => v.split(",")),
    GENERATED_JWT_ISSUER: z.string(),
    GENERATED_JWT_SECONDS_TO_EXPIRE: z.coerce.number(),
```